src/meshes/p4est_mesh.jl

```diff
@@ -517,6 +572,242 @@ function connectivity_structured(n_cells_x, n_cells_y, n_cells_z, periodicity)
 end
 
 
+function connectivity_cubed_sphere(trees_per_face_dimension, layers)
+  n_cells_x = n_cells_y = trees_per_face_dimension
+  n_cells_z = layers
+
+  linear_indices = LinearIndices((trees_per_face_dimension, trees_per_face_dimension, layers, 6))
+
+  # Vertices represent the coordinates of the forest. This is used by p4est
+  # to write VTK files.
+  # Trixi doesn't use p4est's coordinates, so the vertices can be empty.
+  n_vertices = 0
+  n_trees = 6 * n_cells_x * n_cells_y * n_cells_z
```

Comment:
It's just a suggestion: Maybe you can define a variable, e.g., `dims = 3` or `NDIMS = 3` somewhere in your function, and then use that variable everywhere where you syntactically use the number of dimensions. This makes it more descriptive in many ways instead of using "magic constants", e.g., `dims` vs. `3` or `2 * dims` vs. `6` or `2^dims` vs. `8`.

Reply:
While I generally agree to that, I find it more readable in this case to just write 6 as "number of sides of a cube".